nuget/lib/dependabot/nuget/file_parser.rb

```diff
@@ -70,7 +70,7 @@ def project_file_parser
       end
 
       def project_files
-        dependency_files.select { |df| df.name.match?(/\.[a-z]{2}proj$/) }
+        dependency_files.select { |df| df.name.match?(/\.[a-z]{2}proj$|[Dd]irectory.[Pp]ackages.props/) }
```

Comment:
Could we split this regex into two different captures and try to match against both?

Something like 

```suggestion
        projfile = /\.[a-z]{2}proj$/
        packageprops = /[Dd]irectory.[Pp]ackages.props/
        
        dependency_files.select { |df| 
          df.name.match?(projfile) ||  
          df.name.match?(packageprops)
        }
```

and do the same in the file updater?